modules/rosa-create-cluster-admins.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-create-dedicated-cluster-admins.adoc"]
+= Granting cluster-admin access
+
+As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. These privileges are not automatically assigned to your user account when you create the cluster.
+
+Additionally, only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges. As a best practice, limit the number of `cluster-admin` users.
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
+* You have the IDP user name for the user you are creating.
+* You are logged in to the cluster.
+
+.Procedure
+
+. Enable `cluster-admin` capability on the cluster:
++
+[source,terminal]
+----
+$ rosa edit cluster <cluster_name> --enable-cluster-admins
+----
++
+. Give your user `cluster-admin` privileges:
++
+[source,terminal]
+----
+$ rosa grant user cluster-admin --user <idp_user_name> --cluster <cluster_name>
+----
++
+. Verify your user is listed as a `cluster-admin`:
```

Comment:
I would spell out "cluster administrator" here where it is in generic noun form (as opposed to where you use it as an adjective elsewhere, such as _`cluster-admin` privileges_).